lib/vtk/analytics.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'net/http'
+require 'open-uri'
+require 'uri'
+
+module Vtk
+  # Provides command analytics to VTK team
+  class Analytics
+    CLIENT_KEY = '456f76a9dc1a235690017dab981af904'
```

Comment:
Should this API key be stored somewhere more secure?

Reply:
I talked with Joe. We don't like the idea of keeping it here but it's the best we could come up with. It's a client key which aren't meant to be accessible to clients. It's an equivalent of the Google Analytics key that is available in the vets-website repo.

Reply:
Great, thanks for looking into this!